src/lib/selectors.ts

```diff
@@ -586,7 +586,7 @@ export const getDeleteMessage = (state: SendBirdState) => (
             topics.DELETE_MESSAGE,
             { messageId, channel },
           );
-          resolve(message);
+          resolve();
```

Comment:
We dont need to pass down the deleted message?

Reply:
we don't need to, because there's no message of response